src/secp256k1/src/cpp/Scalar.cpp

```diff
@@ -269,10 +269,10 @@ unsigned char* Scalar::deserialize(unsigned char* buffer) {
 
 std::string Scalar::GetHex() const {
     unsigned char buffer[32];
-    std::stringstream ss;
-
     serialize(buffer);
 
+    std::stringstream ss;
+    ss << std::hex;
     for (int i = 0; i < 32; ++i) {
         ss << buffer[i] / 16;
         ss << buffer[i] % 16;
```

Comment:
From my perspective this is hard to read. My idea is we should use hex table instead (like my answer here https://stackoverflow.com/a/16827872/1829232). But maybe it is just only me so if other people think the current way is more better then we should go with it.

Reply:
fixed. but use bare value instead look up hex table.